### DISCUSSION TITLE
Custom overlay icon doesn't work

### DISCUSSION THREAD

**Discussion by jxxxxh** [SaifAqqad/AHK_MicMute#92] [General]
I'm on version 1.2.9 and see that you had a commit to fix for the custom icons. However, I followed the instructions to name the files, but they don't seem to apply. Tried with different icons and restarting program to no avail.

**Comment by SaifAqqad** [comment]
1.2.9 does not include the fix, I will make a new release today.

**Reply by SaifAqqad** [reply]
[1.2.10](https://github.com/SaifAqqad/AHK_MicMute/releases/tag/1.2.10) is out now, but it might take a day for it to be available in scoop though.

**Reply by jxxxxh** [reply]
Awesome, thanks for an awesome product and such quick turnaround! 

I like this product much more than the other popular MicSwitch project on Github. The custom logo was the single issue holding me back from switching over. Thanks again.
